tests/serverpod_test_server/lib/src/custom_classes.dart

```diff
@@ -31,20 +31,82 @@ class CustomClass2 {
   dynamic toJson() => {'text': value};
 }
 
-class CustomClass3 implements SerializableModel, ProtocolSerialization {
-  final String value;
+/// Custom class that does not implement ProtocolSerialization.
```

Comment:
Suggestion: Remove redundant comment.

This is already implied by the name of this and the next class but you can keep the comment for clarification on the last one.